tests/Fixer/Strict/DeclareStrictTypesFixerTest.php

```diff
@@ -141,11 +155,11 @@ public function provideMessyWhitespacesCases()
     {
         return [
             [
-                "<?php declare(strict_types=1);\r\nphpinfo();",
+                "<?php\r\ndeclare(strict_types=1);\r\n\tphpinfo();",
```

Comment:
> you must not change default behaviour of fixer, it will fail on any integration usage that are using this rule.

Oh, I didn't realise that. Should I bug this then?
https://travis-ci.org/Parsemd/Parsemd/builds/249843237
vs.
https://travis-ci.org/Parsemd/Parsemd/builds/250188863
(update killed the cron)

I'll change the default on the position fixer to `same` in that case.